BUILDING.md

```diff
@@ -163,7 +163,8 @@ $ [sudo] make install
 
 Prerequisites:
 
-* [Python 2.6 or 2.7](https://www.python.org/downloads/)
+* [Python 2.6 or 2.7](https://www.python.org/downloads/)  
+  *Note*: It is recommanded to include the "Add python.exe to path" option
```

Comment:
Personally I don't like installers, so I add it to `%Path%` manually.
Also there's [PEP-0514](https://www.python.org/dev/peps/pep-0514/) (https://github.com/nodejs/node/issues/13882) which I also do manually 😊